Weaver/RealmWeaver.Fody/ModuleWeaver.cs

```diff
@@ -548,6 +587,68 @@ private void ReplaceListGetter(PropertyDefinition prop, FieldReference backingFi
         Debug.Write("[get list] ");
     }
 
+        // WARNING
+    // This code setting the backing field only works if the field is settable after init
+    // if you don't have an automatic set; on the property, it shows in the debugger with
+    //         Attributes    Private | InitOnly    Mono.Cecil.FieldAttributes
+    private void ReplaceBacklinksGetter(PropertyDefinition prop, FieldReference backingField, string columnName, TypeReference elementType)
+    {
+        // A synthesized property getter looks like this:
+        //   0: ldarg.0  // load the this pointer
+        //   1: ldfld <backingField>
+        //   2: ret
+        // We want to change it so it looks somewhat like this, in C#
+        /*
+            if (<backingField> == null)
+            {
+               if (IsManaged)
+                     <backingField> = GetBacklinks<T>(<columnName>);
+               else
+                     <backingField> = new Enumerable.Empty<T>.AsQueryable();
+            }
+            // original auto-generated getter starts here
+            return <backingField>; // supplied by the generated getter OR RealmObject._CopyDataFromBackingFields
+        */
+
+        var start = prop.GetMethod.Body.Instructions.First();  // this is a label for return <backingField>;
+        var il = prop.GetMethod.Body.GetILProcessor();
+
+        il.InsertBefore(start, il.Create(OpCodes.Ldarg_0));  // this for field ref [ -> this]
+        il.InsertBefore(start, il.Create(OpCodes.Ldfld, backingField)); // [ this -> field]
+        il.InsertBefore(start, il.Create(OpCodes.Ldnull)); // [field -> field, null]
+        il.InsertBefore(start, il.Create(OpCodes.Ceq));  // [field, null -> bool result]
```

Comment:
Isn't null falsy for IL? I thought we could go for `Brfalse` without explicitly checking for null equality.